src/components/ChatBotBody/ChatBotBody.tsx

```diff
@@ -273,9 +283,9 @@ const ChatBotBody = ({
 						className={`rcb-bot-message ${botBubbleEntryStyle}`}
 					>
 						<div className="rcb-typing-indicator">
```

Comment:
Styles have been added for `rcb-dot`, but it'll be good to have one for `rcb-typing-indicator` as well 😊